conference.js

```diff
@@ -801,11 +804,27 @@ export default {
      * @param {string} id participant's identifier(MUC nickname)
      *
      * @returns {boolean|null} true if participant's connection is ok or false
-     * if the user is having connectivity issues.
+     * if the user is having connectivity issues. Returns null if there is no
+     * such participant.
      */
     isParticipantConnectionActive (id) {
         let participant = this.getParticipantById(id);
-        return participant ? participant.isConnectionActive() : null;
+        return participant
```

Comment:
Now the description of this method is confusing. Given we have more states now, being not in ACTIVE does mean that the user is not connected.